indico/web/client/js/utils/date_format.js

```diff
@@ -24,3 +24,5 @@ export function formatDate(formatString, date) {
     .replace(/m{1,2}/i, s => month.padStart(s.length, '0'))
     .replace(/d{1,2}/i, s => day.padStart(s.length, '0'));
 }
+
+formatDate.ISO_FORMAT = 'yyyy-mm-dd';
```

Comment:
hm why not just `export const ISO_FORMAT = ...` and import it? feels a bit dirty to set this on the function tbh